CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@
     * [Override context directory](#override-context-directory)
     * [Setup test users](#setup-test-users)
     * [Automated tests](#automated-tests)
+    * [Troubleshooting](#troubleshooting-1)
```

Comment:
what's the -1 for?

Reply:
There's 2 `Troubleshooting` sections. Without the `-1`, it takes you to first one:

https://github.com/DataBiosphere/terra-cli/blob/main/CONTRIBUTING.md#troubleshooting